neofetch

```diff
@@ -43,7 +43,7 @@ get_os() {
         "Haiku") os="Haiku" ;;
         "MINIX") os="MINIX" ;;
         "AIX") os="AIX" ;;
-        "IRIX64") os="IRIX" ;;
+        "IRIX64" | "IRIX") os="IRIX" ;;
```

Comment:
For simpler detection just change both `"IRIX64" | "IRIX")` to `"IRIX"*)`.

Reply:
Fair enough, unlikely to be some other kernel named irix-something